src/scss/utilities/_sizing.scss

```diff
@@ -0,0 +1 @@
+.mw-0 { min-width: 0!important; }
```

Comment:
Since Bootstrap has `mw-100` where `mw` stands for `max-width` I think this could potentially cause some confusion.

Reply:
Fair. Can we propose a better name? `min-width-0`?

Reply:
There is `min-vw-100` for `min-width: 100vw` so maybe we could do `min-w-0`?

Reply:
Okay will do.